Frameworks/CoreData/NSPropertyDescription.mm

```diff
@@ -14,10 +14,64 @@
 //
 //******************************************************************************
 
+#import <CoreData/CoreData.h>
+#import <Foundation/Foundation.h>
 #import <StubReturn.h>
-#import <CoreData/NSPropertyDescription.h>
+
+#import "NSManagedObjectModel-XMLParsing.h"
 
 @implementation NSPropertyDescription
+- (id)initWithXMLElementName:(NSString*)entityName attributes:(NSDictionary<NSString*, NSString*>*)attributes {
```

Comment:
Alas, this is for _CDXMLCoding protocol conformance. Just because this class does not use it, it doesn't mean that other protocol consumers won't.

---
In reply to: [69757738](https://github.com/Microsoft/WinObjC/pull/610#discussion_r69757738) [](ancestors = 69757738)